problems/removeNumbersAtOddIndices.js

```diff
@@ -2,8 +2,12 @@
 * @param {number[]} arr - The input array
 * @returns {number[]} - An array removing all elements initially appearing at an odd index
 */
-function removeNumbersAtOddIndices() {
-
+function removeNumbersAtOddIndices(arr) {
+    let newArr = [];
+    for (let i = 0; i < arr.length; i += 2) {
```

Comment:
Try using .filter for cleaner code
Remember the callback can take in an index as the second argument